ansible_builder/main.py

```diff
@@ -38,30 +37,21 @@
 class AnsibleBuilder:
     def __init__(self, action=None,
                  filename=constants.default_file,
-                 base_image=None,
+                 build_args=None,
                  build_context=constants.default_build_context,
                  tag=constants.default_tag,
                  container_runtime=constants.default_container_runtime,
                  verbosity=2):
         self.action = action
         self.definition = UserDefinition(filename=filename)
 
-        # Handle precedence of the base image
-        if base_image is not None:
-            self.base_image = base_image
-        if base_image is None:
-            if self.definition.raw.get('base_image'):
-                self.base_image = self.definition.raw.get('base_image')
```

Comment:
@beeankha I hat to just remove a feature, but if we're not taking the base image as a CLI option, then we shouldn't take it in the definition either.

If we still want some kind of support for this, I would suggest that we should support build args in the definition.

If a user gives a dict of build args in the definition file, then maybe those should become defaults in the Containerfile? I'm still noodling on that idea.